app/(auth)/verify-popover.tsx

```diff
@@ -0,0 +1,61 @@
+import { router, useNavigation } from 'expo-router';
+import React, { useContext, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import VerificationPopover from '@/components/ui/popover/VerificationPopover';
+import { VerificationContext } from '@/context/auth/VerificationContext';
+import useSendVerificationEmail from '@/hooks/auth/UseSendVerificationEmail';
+import useVerify from '@/hooks/auth/UseVerify';
+
+export default function VerifyPopover() {
+  const { t } = useTranslation();
+  const navigation = useNavigation();
+  const { verificationProps, setVerificationProps } = useContext(VerificationContext);
+  const [error, setError] = useState(false);
+  const { mutate, isPending, isError } = useVerify(verificationProps.email, verificationProps.code);
+
+  const { mutate: mutate2, isPending: isPending2 } = useSendVerificationEmail(
```

Comment:
maybe we name it after the hook name without using mutate everywere ?

Reply:
done